search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -0,0 +1,223 @@
+    (core-api/search-response ctx results)))
+
+(defn- find-tiles
+  "Retrieves all the tiles which intersect the input geometry"
+  [ctx params]
+  (let [results (query-svc/find-tiles-by-geometry ctx params)]
+    {:status 200
+     :headers {common-routes/CONTENT_TYPE_HEADER (mt/with-utf-8 mt/json)}
+     :body results}))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Route Definitions
+
+(defn- get-deleted-collections
+  "Invokes query service to search for collections that are deleted and returns the response"
+  [ctx path-w-extension params headers]
+  (let [params (process-params :collection params path-w-extension headers mt/xml)]
+    (info (format "Searching for deleted collections from client %s in format %s with params %s."
+                  (:client-id ctx) (rfh/printable-result-format (:result-format params))
+                  (pr-str params)))
+    (core-api/search-response ctx (query-svc/get-deleted-collections ctx params))))
+
+(defn- get-deleted-granules
+  "Invokes query service to search for granules that are deleted and returns the response"
+  [ctx path-w-extension params headers]
+  (let [params (process-params nil params path-w-extension headers mt/xml)]
+    (info (format "Searching for deleted granules from client %s in format %s with params %s."
+                  (:client-id ctx) (rfh/printable-result-format (:result-format params))
+                  (pr-str params)))
+    (core-api/search-response ctx (query-svc/get-deleted-granules ctx params))))
+
+(def search-routes
+  (context ["/:path-w-extension" :path-w-extension #"(?:(?:granules)|(?:collections)|(?:variables)|(?:services))(?:\..+)?"] [path-w-extension]
+    (OPTIONS "/" req common-routes/options-response)
+    (GET "/"
+      {params :params headers :headers ctx :request-context query-string :query-string}
+      ;; XXX REMOVE this check and the stubs once the service and
+      ;;     the associations work is complete
+      (if (headers "cmr-prototype-umm")
+        (stubs/handle-prototype-request
+         path-w-extension params headers query-string)
+        (find-concepts ctx path-w-extension params headers query-string)))
+    ;; Find concepts - form encoded or JSON
+    (POST "/"
+      {params :params headers :headers ctx :request-context body :body-copy}
+      (find-concepts ctx path-w-extension params headers body))))
+
+(def granule-timeline-routes
+  (context ["/granules/:path-w-extension" :path-w-extension #"(?:timeline)(?:\..+)?"] [path-w-extension]
+    (OPTIONS "/" req common-routes/options-response)
+    (GET "/"
+      {params :params headers :headers ctx :request-context query-string :query-string}
+      (get-granules-timeline ctx path-w-extension params headers query-string))
+    (POST "/" {params :params headers :headers ctx :request-context body :body-copy}
+      (get-granules-timeline ctx path-w-extension params headers body))))
+
+(def deleted-routes
```

Comment:
This var name makes me think these routes have been deleted. Maybe find-deleted-concepts-routes?

Reply:
Good call.

Reply:
Fixed.